db/seeds.rb

```diff
@@ -87,7 +87,7 @@ def load_test_users
       suburb_file.pos = 0 if suburb_file.eof?
       row = CSV.parse(suburb_file.readline)
 
-      suburb, country, state, latitude, longitude = row[0]
+      suburb, _country, _state, latitude, longitude = row[0]
```

Comment:
@maco: yep, "variable starting with underscore" is a fairly widespread convention for either "private variable" or "variable that can be ignored".

@Br3nda: Good question. This code was merged in #731, and the discussion there doesn't explain where the file `suburbs.csv` came from. It looks like it's a list of Australian suburbs generated for some other purpose and included wholesale - an Australian government database, perhaps? We should probably know more about its origin, to check we're not causing any licensing issues. @oshiho3, where did that file come from?